docs/KNOWNISSUES.md

```diff
@@ -2,138 +2,120 @@
 
 ## Known Issues for PowerShell on Non-Windows Platforms
 
-The first Alpha release of PowerShell on Linux and macOS is mostly functional but
-does have some significant limitations and usability issues.
+Alpha releases of PowerShell on Linux and macOS are mostly functional but do have some significant limitations and usability issues.
+Beta releases of PowerShell on Linux and macOS are more functional and stable than alpha releases, but still may be lacking some set of features, and can contain some bugs.
```

Comment:
If you think you need `some` to clarify `bugs`, then I think the "significant limitations and usability issues"  is clearer.